### PR TITLE
Create writing2.md

### PR DESCRIPTION
**Pull request by olgaz-skiba**

### CODE REVIEW

**Review by VladimirPetranovski — APPROVED** [review]
lgfm

**Review by Alex99905 — APPROVED** [review]
Lgfm

File: homework/english/skiba_olga/lecture2/writing2.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+I have tried to start work in IT since 2014 . 
+Then i went to course of QA Automatisation but I did not have enough interest and knowledge of English in this profession.
```
Comment (LiudmilaZhdanok) [inline homework/english/skiba_olga/lecture2/writing2.md:2]: I think "in that profession" whoud be correctly

File: homework/english/skiba_olga/lecture2/writing2.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+I have tried to start work in IT since 2014 . 
+Then i went to course of QA Automatisation but I did not have enough interest and knowledge of English in this profession. 
+At the end of last year i saw an application of javarash, it interested me.
+After that i started to look for information related to java programming.
+I saw the Javagura course and applied for it.
```
Comment (LiudmilaZhdanok) [inline homework/english/skiba_olga/lecture2/writing2.md:5]: I think "found" would be more correct

**Review by LiudmilaZhdanok — APPROVED** [review]
See notes

**Review by squarple — APPROVED** [review]

**Pull request merged**